### PR TITLE
Add new flex-item, html elements, style for input

### PR DESCRIPTION
**Pull request by AndriySikora**
New style for input, cart element and sign in. New html elements. Created 3 flex-items.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 }
-@media screen and (min-width: 899px) {
+@media screen and (min-width: 1000px) {
   .main-wrapper {
     width: 100%;
   }
 }
 .navigation-bar {
-  height: 33px;
+  height: 100%;
```
Comment (answer261) [inline css/main.css]: do you really need height there? :)

File: css/main.css [hunk 2]
```diff
@@ -93,20 +91,20 @@
...
 .top-level-menu > li:hover {
   background: #f2f2f2;
 }
 .top-level-menu li:hover > ul {
-  display: inline;
+  display: inline-block;
 }
 .third-level-menu a {
-  color: black;
+  color: #505050;
```
Comment (answer261) [inline css/main.css]: make a variable with this color :)

File: index.html [hunk 3]
```diff
@@ -85,21 +85,18 @@
...
-		<div class="flex-item-nav">
-			<div class="shoping">
-				<i class="fa fa-shopping-cart"></i>
-				<span class="shoping-amount">0</span>
-			</div>
-		</div>
-		<div class="flex-item-nav sign-in"><a href="#">Sign in</a>
-		</div> 
-	</div>
+			<div class="flex-item-nav">
```
Comment (answer261) [inline index.html]: logo and menu should also be merged into one flex container

File: css/main.css (hunk 1 above)
Comment (AndriySikora) [inline css/main.css]: no, I don't need height there:-)

File: index.html (hunk 3 above)
Comment (AndriySikora) [inline index.html]: @answer261 I can't move logo in the second flex-item, because of  inside it, it won't work properly after I'll make this.

**Pull request merged**